### ISSUE TITLE
Wanted, clone option in "edit topology"

### ISSUE THREAD

**Issue by DorinevanderVlies** [quintel/etmoses#1499]
When working on a topology it can be useful to use an existing topology to start with. Currently there is no `clone` option when editing a topology. 

Previously this was less necessary as the YAML code of an existing topology could be copied to a new empty topology. With the new interface lay-out (which I really like btw!) this copying is not possible any more.

**Comment by grdw** [comment]
On hold after #1515 is merged.
